### PR TITLE
Remove crawling for unsupported Red Hat kernel versions

### PR DESCRIPTION
**Pull request by robbycochran**
Remove crawling for Red Hat kernel versions from OCP versions that are no longer supported by RHACS

### CODE REVIEW

File: kernel-crawler/kernel-crawler.py [hunk 1]
```diff
@@ -287,8 +287,9 @@
             "type": "s3",
             "root": "https://storage.googleapis.com/cos-tools",
             "patterns": [
-                "\d+\.\d+\.\d+/kernel-src.tar.gz$",
-                "\d+\.\d+\.\d+/kernel-headers\.t(ar\.)?gz$",
+                # Only include kernels that are part of LTS releases https://cloud.google.com/container-optimized-os/docs/release-notes
+                "(16108|16623|16919|17162|17412)\.\d+\.\d+/kernel-src.tar.gz$",
+                "(16108|16623|16919|17162|17412)\.\d+\.\d+/kernel-headers\.t(ar\.)?gz$",
```
Comment (Molter73) [inline kernel-crawler/kernel-crawler.py:292]: Will we have to update this manually when a new version of COS rolls around? I'm not super familiar with their versioning but those numbers look like they correspond to some kind of milestone, so I'd assume the next LTS release will have a different version number.

Alternative take is that we'll only support COS via CO-RE BPF for future LTS versions, on comment clarifying that would be nice if that's the case.

File: kernel-crawler/rhel-subscription-crawler/crawler.py [hunk 2]
```diff
@@ -36,8 +36,7 @@ def __init__(self, offline_token: str, get_latest: bool, rhel_package_lists: str
         ]
 
         self.repo_include_patterns = [
-            re.compile(r'^rhel-[6-8].*$'),
-            re.compile(r'^rhel-8.*$'),
+            re.compile(r'^rhel-[6-9].*$'),
```
Comment (Molter73) [inline kernel-crawler/rhel-subscription-crawler/crawler.py:39]: Do we still support RHEL 6? I'm pretty sure we haven't compile a RHEL 6 driver in a couple years.
```suggestion
            re.compile(r'^rhel-[7-9].*$'),
```

**Review by Molter73 — APPROVED** [review]
LGTM! Just have 2 minor comments.

File: kernel-crawler/kernel-crawler.py (hunk 1 above)
Comment (robbycochran) [reply kernel-crawler/kernel-crawler.py:292]: Good point. Yes, these are LTS release numbers and we'd have to update this for future versions. I think we just need a better filter. I'll remove this for now.

**Pull request merged**